src/go/rpk/pkg/cli/cluster/connections/list.go

```diff
@@ -52,6 +58,17 @@ var tableHeaders = []string{
 	"REQS/MIN",
 }
 
+func isFeatureSupported(ctx context.Context, client *rpadmin.AdminAPI) bool {
+	// Determine whether the broker version supports client connection monitoring
+	brokers, err := client.Brokers(ctx)
+	if err != nil || len(brokers) == 0 {
+		return false
+	}
+
+	brokerVersion, err := redpanda.VersionFromString(brokers[0].Version)
```

Comment:
Just to be safe, we should check for every broker version. I _believe_ RP will fail if one of the brokers is on an older version (failed upgrade state or something like that)